pkg/constants/constants.go

```diff
@@ -44,6 +44,11 @@ var (
 	InferenceServiceConfigMapName = "inferenceservice-config"
 )
 
+// TrainedModel Constants
+var (
+	TrainedModelMemoryIncluded = "trainedmodel-memory-included"
```

Comment:
I think the name is too long, remember that the length limit for label is only 63 chars
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set